.github/ISSUE_TEMPLATE/schema-issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Schema Issue
+about: "Report an issue with the ECS schema."
```

Comment:
```suggestion
about: "Report an issue with ECS."
```

My brother, despite not being a software developer, understood recursion. He would say that DOS stood for the "DOS Operating System" 😂